src/assets/js/scene.js

```diff
@@ -40,6 +42,8 @@ export default class GameScene extends Scene {
     this.bullets = new Bullets(this, this.bulletPositionX, this.bulletPositionY, this.ship);
     this.bulletsGroup = [];
     this.asteroidGroup = [];
+
+    this.blackHole = new Blackhole(this, 100, 100, 10);
```

Comment:
We'll be creating the blackhole in the level generator, so this should be removed.

Reply:
fair enough